pydatalab/src/pydatalab/apps/ftir/__init__.py

```diff
@@ -69,16 +106,22 @@ def _format_ftir_plot(self, ftir_data: pd.DataFrame) -> bokeh.layouts.layout:
             plot_line=True,
             tools=HoverTool(
                 tooltips=[
-                    ("Wavenumber / cm⁻¹", "@Wavenumber{0.00}"),
-                    ("Absorbance", "@Absorbance{0.0000}"),
+                    (toollabsx, "@Wavenumber{0.00}"),
+                    (toollabsy, "@Absorbance{0.0000}"),
                 ],  # Display x and y values to specified decimal places
                 mode="vline",  # Ensures hover follows the x-axis
             ),
         )
-        # Adding cm^-1 to the x-axis label using unicode characters - might be a more logical way
-        layout.children[1].xaxis.axis_label = "Wavenumber / cm⁻¹"
+        if xunits == 'default':
+            # Adding cm^-1 to the x-axis label using unicode characters - might be a more logical way
+            layout.children[1].xaxis.axis_label = "Wavenumber / cm⁻¹"
+        else:
```

Comment:
I just made this change so that the label appears the same with a superscript as in the other mode.

```suggestion
        elif xunits == "1/CM\n":
            layout.children[1].xaxis.axis_label = "Wavenumber / cm⁻¹"
        else:
```